### PR TITLE
[Synapse] Updated help doc for [Set-AzSynapseLinkConnection] and [Set-AzSynapseLinkConnectionLinkTable]

### PR DESCRIPTION
**Pull request by yanjungao718**
Updated help doc for **Set-AzSynapseLinkConnection** and **Set-AzSynapseLinkConnectionLinkTable**, add sample json file to these two cmdlets.

## Checklist

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from the scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** change version of module in pull request

### CODE REVIEW

File: src/Synapse/Synapse/help/Set-AzSynapseLinkConnection.md [hunk 1]
```diff
@@ -31,6 +31,37 @@ The **Set-AzSynapseLinkConnection** cmdlet creates or updates a link connections
...
+			}
+		},
+		"compute":{
+			"coreCount":16,
+			"computeType":"General"
+		}
+	}
+}
+#>
 Set-AzSynapseLinkConnection -WorkspaceName ContosoWorkspace -DefinitionFile "C:\\samples\\linkconnection.json"
```
Comment (isra-fel) [inline src/Synapse/Synapse/help/Set-AzSynapseLinkConnection.md:65]: The json part looks good. Problem is in PS back-slash ("\") doesn't need to be escaped. You can simply use `C:\samples\linkconnection.json`.

File: src/Synapse/Synapse/help/Set-AzSynapseLinkConnectionLinkTable.md [hunk 2]
```diff
@@ -39,6 +39,29 @@ The **Set-AzSynapseLinkConnectionLinkTables** cmdlet edits link tables under a l
...
+          "type": "Round_RoBin", // please choose a type from Hash, Round_RoBin, Replicate
+          "distributionColumn": "sampleColumn" // please change to the column name
+        }
+      }, 
+      "operationType": "add" // please choose a value from add, update, remove
+    }
+  ]
+}
+#>
 Set-AzSynapseLinkConnectionLinkTable -WorkspaceName ContosoWorkspace -LinkConnectionName ContosoLinkConnection -EditTablesRequestFile "C:\\samples\\edittables.json"
```
Comment (isra-fel) [inline src/Synapse/Synapse/help/Set-AzSynapseLinkConnectionLinkTable.md:65]: Same here

**Review by isra-fel — CHANGES_REQUESTED** [review]

File: src/Synapse/Synapse/help/Set-AzSynapseLinkConnection.md (hunk 1 above)
Comment (yanjungao718) [reply src/Synapse/Synapse/help/Set-AzSynapseLinkConnection.md]: Modified, thanks!

File: src/Synapse/Synapse/help/Set-AzSynapseLinkConnectionLinkTable.md (hunk 2 above)
Comment (yanjungao718) [reply src/Synapse/Synapse/help/Set-AzSynapseLinkConnectionLinkTable.md]: Done, thanks!

**Review by isra-fel — APPROVED** [review]
LGTM

**Pull request merged**